compiler/src/iree/compiler/Codegen/Common/test/bufferize_copy_only_dispatches.mlir

```diff
@@ -86,3 +86,42 @@ builtin.module {
 //       CHECK:   linalg.generic
 //  CHECK-SAME:       ins(%[[ZERO]] :
 //  CHECK-SAME:       outs(%[[SUBVIEW]] :
+
+// -----
+
+#map = affine_map<(d0) -> (d0)>
```

Comment:
can you trim down the test case a bit? the more IR we have like this the harder it is to refactor things - this could probably just be @concatenate_cst like above but with a linalg.fill of the subspan value instead of the flow.dispatch.tensor.store

Reply:
I need the allocation for it to actually test something. I can try to come up with a better way to do it though.

Reply:
@benvanik lmk if the most recent update works.